lib/extensions/simple_form/components/submit_component.rb

```diff
@@ -0,0 +1,10 @@
+module SubmitComponent
+  def submit(*args, &block)
```

Comment:
What if we called it `submit_button`? Seeing `f.submit` everywhere in templates made it seem more imperative, but that could be just be me

Reply:
This is overriding an existing method (the [`submit` FormBuilder method](https://api.rubyonrails.org/v6.1.0/classes/ActionView/Helpers/FormBuilder.html#method-i-submit)), which we could have been using prior to these changes (and did in a [couple places](https://github.com/18F/identity-idp/blob/cfbbcf7ad39e8034dfede525642c46e2e5e4985d/app/views/users/piv_cac_authentication_setup/new.html.erb#L34)). The changes from `f.button :submit` to `f.submit` were not strictly necessary, but I introduced them for brevity and consistency.

Reply:
ah I see, and I guess there's no reason we'd ever want to use the original version right?

Reply:
I can't imagine so, and part of the intention here is to replace those defaults with our desired behavior.

But this reminds me I wanted to highlight a choice we'd have as far as opting for the SimpleForm DSL or the ViewComponent rendering API, since these would largely be equivalent:

```erb
<%= f.submit 'Submit' %>
```

```erb
<%= render SubmitButtonComponent.new.with_content('Submit') %>
```

I see pros and cons to both (in terms of compactness, explicitness). Using the component API would also avoid having to override the default form builder method. But, again, I doubt we'd need the default.